bin/accessibility-automation/cypress/index.js

```diff
@@ -0,0 +1,113 @@
+/* Event listeners + custom commands for Cypress */
+
+Cypress.on('test:before:run', () => {
+  if (Cypress.env("IS_ACCESSIBILITY_EXTENSION_LOADED") !== "true") return
+  const extensionPath = Cypress.env("ACCESSIBILITY_EXTENSION_PATH")
+
+  if (extensionPath !== undefined) {
+    new Promise((resolve, reject) => {
+      window.parent.addEventListener('A11Y_TAP_STARTED', () => {
+        resolve("A11Y_TAP_STARTED");
+      });
+      const e = new CustomEvent('A11Y_FORCE_START');
+      window.parent.dispatchEvent(e);
+    })
+  }
+})
+
+Cypress.on('test:after:run', (attributes, runnable) => {
```

Comment:
Wrap everything under this callback under try catch else user tests would break

Reply:
done